content/about/feature-stages/index.md

```diff
@@ -85,8 +74,8 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Beta
-| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Beta
+| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Stable
+| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Stable
```

Comment:
Is this really stable since we have helm as beta down below and helm is our recommended way for 2 of our install options?